backend/news_api_scraper.py

```
@@ -0,0 +1,495 @@
+                    value_score, competition_score, total_score, created_at, updated_at,
+                    industry_category, source_reliability_score, last_scraped_at
+                ) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)
+            """, (
+                opp.title, description[:2000], opp.source_name, f"NEWS-{hash(opp.duplicate_hash) % 100000}",
+                None, opp.published_at, None, 'news_api', f"NewsAPI-{opp.source_name}",
+                opp.url, None, opp.author, relevance_score, urgency_score,
+                value_score, competition_score, total_score, now, now,
+                opp.opportunity_type, 0.7, now  # News has lower reliability than direct sources
+            ))
+            
+            opportunity_id = cursor.lastrowid
```

Ruff (0.11.9)

Local variable `opportunity_id` is assigned to but never used

Remove assignment to unused variable `opportunity_id`

(F841)